patches/sitecustomize.py

```diff
@@ -1,22 +1,56 @@
 # TODO(rosbo): Remove this once we fix the issue with fastai importing older libcudnn if imported prior to tensorflow
 import tensorflow
 import os
+from google.auth import credentials
```

Comment:
These imports are no longer conditional on `KAGGLE_DATA_PROXY_TOKEN` being set.  This could be detrimental for non-Kaggle kernels use of our Docker image.  I suppose only startup latency of any Python binary, but there could be more to it than that.  If you do plan for it, suggest to blame back and find from the author why that approach was taken (hopefully that doesn't circle back to me ;) ).

Reply:
ping

Reply:
Sorry I hadn't finished addressing everything yet :) I wish Github PRs were more like critique and allowed setting the attention set only once you're done.

Reply:
I can make all of this conditional on (KAGGLE_DATA_PROXY_TOKEN || KAGGLE_BQ_USER_JWT) ?

Reply:
SGTM